lib/project_types/script/forms/create.rb

```diff
@@ -28,6 +29,23 @@ def valid_name
         return n if n.match?(/^[0-9A-Za-z_-]*$/)
         raise Errors::InvalidScriptNameError
       end
+
+      def ask_language
+        all_languages = Layers::Application::SupportedLanguages.all
+
+        if language
+          requested = all_languages.find { |l| l == language }
```

Comment:
What do you think of improving this a bit by:

1. Defining a `find` method in the `SupportedLanguages` class, to delegate the responsibility completely
2. When finding the language, normalizing the language string (downcasing it) so that devs can pass in `assemblyscript` too instead of only `AssemblyScript` . Arguably if we only want to allow one, I think it should be always in lowercase.